### ISSUE TITLE
Skill API is not working

### ISSUE THREAD

**Issue by redar84** [workforce-data-initiative/skills-api#34]
I used both postman and curl to get the skills, but the API is not working. Is there any other way to get that skill sets data. Thanks

**Comment by sqpdln** [comment]
The API still seems to be down...

**Comment by JulioCMR** [comment]
Over two year and still not working, maybe RIP

**Comment by parker-howell** [comment]
still down? RIP

**Comment by jsonkcli** [comment]
DId anyone find an alternative?

**Comment by adelucca** [comment]
RIP :( 

Also interested in alternative

**Comment by jsonkcli** [comment]
> RIP :(
> 
> Also interested in alternative

scraped linkedIn for the better

**Comment by adelucca** [comment]
> > RIP :(
> > Also interested in alternative
> 
> scraped linkedIn for the better

Yeah, I ended up scraping a few career blogs that had job titles and skills. Ty
